pkg/timeutils/utils.go

```diff
@@ -9,12 +9,8 @@ import (
 // or Unix timestamp, if successful returns a Unix timestamp
 // as string otherwise returns value back.
 func GetTimestamp(value string) string {
-	format := RFC3339NanoFixed
 	loc := time.FixedZone(time.Now().Zone())
-	if len(value) < len(format) {
```

Comment:
@duglin first of all this code was not called by the logging or anything like that.

So this code might be fixing problems w/ `RFC3339NanoFixed` in the past but since that's gone that's not needed.